phpstan.neon.dist

```diff
@@ -1,23 +1,26 @@
 includes:
-	- phpstan-baseline.neon
+    - phpstan-baseline.neon
 
 rules:
     - Symplify\PHPStanRules\Rules\Explicit\NoMixedPropertyFetcherRule
     - Symplify\PHPStanRules\Rules\Explicit\NoMixedMethodCallerRule
 
 parameters:
-	level: 8
-	checkMissingIterableValueType: false
-	checkGenericClassInNonGenericObjectType: false
-	treatPhpDocTypesAsCertain: false
-	bootstrapFiles:
-		- tests/bootstrap.php
-	paths:
-		- src/
+    level: 8
+    treatPhpDocTypesAsCertain: false
+    bootstrapFiles:
+        - tests/bootstrap.php
+    paths:
+        - src/
+    ignoreErrors:
+        -
+          identifier: missingType.iterableValue
+        -
+          identifier: missingType.generics
 
 services:
-	-
-		class: Cake\PHPStan\AssociationTableMixinClassReflectionExtension
-		tags:
-			- phpstan.broker.methodsClassReflectionExtension
-			- phpstan.broker.propertiesClassReflectionExtension
+    -
+        class: Cake\PHPStan\AssociationTableMixinClassReflectionExtension
+        tags:
+            - phpstan.broker.methodsClassReflectionExtension
+            - phpstan.broker.propertiesClassReflectionExtension
```

Comment:
I think @ADmad made these tabs (default neon) for a reason, but maybe not.

Reply:
I'll revert the tab/space swap.